scripts/filetypes.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
Shellcheck OK
Looks like we do need bash (not /bin/sh) for the pattern matching.